code/package.json

```diff
@@ -15,8 +15,8 @@
     "compile": "electron-webpack",
     "dist": "yarn compile && electron-builder",
     "dist:dir": "yarn dist --dir -c.compression=store -c.mac.identity=null",
-    "web-dev": "webpack-dev-server --hot --open --config webpack.web.config.js --mode development",
```

Comment:
webpack-cli got updated or something, and now it wants a slightly different syntax. \shrug